java/SerializationUtilities/template.yaml

```
@@ -0,0 +1,63 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  serialization utils demo
+
+Globals:
+  Function:
+    Timeout: 20
+    Runtime: java11
+    MemorySize: 512
+    Tracing: Active
+
+
+Resources:
+  APIGatewayDes erializationFunction:
```

```suggestion
  APIGatewayDeserializationFunction:
```
there's a space in the name